ui/app/src/Utils/DateTimePicker/Types.elm

```diff
@@ -0,0 +1,77 @@
+module Utils.DateTimePicker.Types exposing
+    ( DateTimePicker
+    , InputHourOrMinute(..)
+    , Msg(..)
+    , PickerConfig
+    , StartOrEnd(..)
+    , defaultPickerConfig
+    , initDateTimePicker
+    , initFromStartAndEndTime
+    )
+
+import Time exposing (Posix, Zone)
+
+
+type alias DateTimePicker =
+    { month : Maybe Posix
+    , mouseOverDay : Maybe Posix
+    , startDate : Maybe Posix
+    , endDate : Maybe Posix
+    , startTime : Maybe Posix
+    , endTime : Maybe Posix
+    }
+
+
+type alias PickerConfig msg =
+    { zone : Zone
```

Comment:
I think it is safe to assume that this is always going to be in `utc`. You can simplify a lot of code and just use `utc` everywhere.